src/agent/src/image_rpc.rs

```diff
@@ -42,6 +42,7 @@ const UMOCI_PATH: &str = "/usr/local/bin/umoci";
 const IMAGE_OCI: &str = "image_oci";
 const AA_PATH: &str = "/usr/local/bin/attestation-agent";
 const AA_PORT: &str = "127.0.0.1:50000";
+const RESOURCE_PORT: &str = "127.0.0.1:50001";
```

Comment:
we can prefix with `AA_` to keep consistent with other AA configs, and `AA_PORT` can also change to `AA_KEYPROVIDER_PORT` ?